images/macos/macos-11-Readme.md

```diff
@@ -96,22 +94,23 @@
 - zstd 1.5.2
 
 ### Tools
-- Aliyun CLI 3.0.123
-- App Center CLI 2.10.10
-- AWS CLI 2.7.21
-- AWS SAM CLI 1.53.0
+- Aliyun CLI 3.0.124
+- App Center CLI 2.11.0
+- AWS CLI 2.7.23
+- AWS SAM CLI 1.54.0
 - AWS Session Manager CLI 1.2.339.0
 - Azure CLI (azure-devops) 0.25.0
 - Azure CLI 2.39.0
 - Bicep CLI 0.9.1
 - Cabal 3.6.2.0
 - Cmake 3.24.0
-- Fastlane 2.208.0
-- GHC 9.2.4
+- CodeQL Action Bundle 2.10.2
```

Comment:
Should we make sure that https://github.com/actions/runner-images/pull/6092 works first?

Reply:
Why we should wait #6092 if we successfully got this info?

Reply:
tests are running for #6092 , but i agree with @al-cheb